packages/twenty-front/src/modules/settings/security/hooks/__tests__/useCreateSSOIdentityProvider.test.tsx

```diff
@@ -0,0 +1,92 @@
+import { renderHook } from '@testing-library/react';
+import { ReactNode } from 'react';
+import { RecoilRoot } from 'recoil';
+
+import { useCreateSSOIdentityProvider } from '@/settings/security/hooks/useCreateSSOIdentityProvider';
+
+const mutationOIDCCallSpy = jest.fn();
+const mutationSAMLCallSpy = jest.fn();
+
+jest.mock('~/generated/graphql', () => ({
```

Comment:
It's fine to use jest.mock but we prefer more accurate mocks see useRightDrawerEmailThread for example. It's fine here, just FYI

Reply:
I understand, but why do you consider it more accurate given that graphql.tsx is generated? What type of code is being tested that isn't evaluated using this method?